src/domain/reservation/manage/page/ManageReservationsPage.js

```diff
@@ -127,20 +139,52 @@ class ManageReservationsPage extends React.Component {
 
   onEditReservation = (reservation, status) => {
     if (status === RESERVATION_STATE.CANCELLED) {
-      cancelReservation(reservation).then(() => this.loadReservations());
+      reservationUtils.cancelReservation(reservation).then(() => this.loadReservations());
     } else {
-      putReservation(reservation, { state: status }).then(() => {
+      reservationUtils.putReservation(reservation, { state: status }).then(() => {
         this.loadReservations();
       });
     }
   }
 
   onSaveComment = (reservation, comments) => {
-    return putReservation(reservation, { resource: reservation.resource.id, comments }).then(() => {
+    return reservationUtils.putReservation(reservation, { resource: reservation.resource.id, comments }).then(() => {
       this.loadReservations();
     });
   };
 
+  /**
+   * Filter list of reservations based on selected SHOW_ONLY filters.
+   * By default return all fetched reservations
+   *
+   * @memberof ManageReservationsPage
+  */
+
+  getFilteredReservations = (filters) => {
+    const { reservations } = this.state;
+    const { userFavoriteResources } = this.props;
+
+    const favoriteResourceFilter = reservation => userFavoriteResources
+      && userFavoriteResources.includes(reservation.resource.id);
+
+    const canModifyFilter = reservation => reservationUtils.canUserModifyReservation(reservation);
+
+    // Both options selected
+    if (filters.length > 1) {
+      return reservations.filter(favoriteResourceFilter && canModifyFilter);
```

Comment:
If I'm reading this correctly, only `canModifyFilter` is used for filtering. Is this intended?

Reply:
yeah it is, cause the button label is the same, so that filter also reflect that.

P/s: can_modify and can_delete was one in respa (check the sourcecode)

Reply:
Instead of making 2 duplicated line, i make it as a var :P

Reply:
Fixed